src/app/location/location.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-location',
+  templateUrl: './location.component.html',
+  styleUrls: ['./location.component.css']
+})
+export class LocationComponent implements OnInit {
+  locations = [
```

Comment:
we should update this with the complete list at some point sooner rather than later